deploy/kube-config/influxdb/grafana-service.yaml

```diff
@@ -2,16 +2,17 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
```

Comment:
Made all services `cluster-service` and added notes on running them independently.